tensorbay/exception.py

```diff
@@ -91,29 +91,34 @@ class ResponseError(TensorBayClientException):
     Arguments:
         response: The response of the request.
 
+    Attributes:
+        response: The response of the request.
+
     """
 
+    # https://github.com/python/mypy/issues/6473
+    # https://github.com/PyCQA/pyflakes/issues/259
+    # https://github.com/PyCQA/pyflakes/pull/588
+    _INDENT = " " * len(__qualname__)  # type: ignore[name-defined]  # noqa F821
```

Comment:
Why use `__qualname__` here?

Reply:
To acquire the name of the current class.